root/edit/details/AddRelease.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+import ReleaseEvents
+  from '../../static/scripts/common/components/ReleaseEvents';
+import formatBarcode from '../../static/scripts/common/utility/formatBarcode';
+
+type AddReleaseEditT = {
+  ...EditT,
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +barcode: string | null,
+    +comment: string,
+    +events?: $ReadOnlyArray<ReleaseEventT>,
+    +language: LanguageT | null,
+    +name: string,
+    +packaging: ReleasePackagingT | null,
+    +release: ReleaseT,
+    +release_group: ReleaseGroupT,
+    +script: ScriptT | null,
+    +status: ReleaseStatusT | null,
+  },
+};
+
+type Props = {
+  +allowNew?: boolean,
+  +edit: AddReleaseEditT,
+};
+
+const AddRelease = ({allowNew, edit}: Props): React.MixedElement => {
+  const display = edit.display_data;
+  const language = display.language;
+  const packaging = display.packaging;
+  const script = display.script;
+  const status = display.status;
+
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Release'))}</th>
+            <td>
+              <DescriptiveLink
+                allowNew={allowNew}
+                entity={display.release}
+              />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <table className="details add-release">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Name'))}</th>
+            <td>{display.name}</td>
+          </tr>
+
+          <tr>
+            <th>{addColonText(l('Artist'))}</th>
+            <td>
+              <ExpandedArtistCredit
+                artistCredit={display.artist_credit}
+              />
+            </td>
+          </tr>
+
+          <tr>
+            <th>{addColonText(l('Release Group'))}</th>
```

Comment:
The old template has group lowercased (Release group), which is consistent with "Release events"

Reply:
I copied this from AddReleaseGroup, but most edits seem to use "Release group" so I changed the three that did not in a second commit.